runtime-image/bin/Dockerfile.gen_keys

```diff
@@ -1,4 +1,6 @@
-FROM debian:8
+FROM gcr.io/google-appengine/debian9
```

Comment:
We should actually have this use the same Ubuntu image that the runtime image uses.  When I wrote this file, the runtime image was still using debian 8.

Reply:
Switched to ubuntu.